js/src/sdk/models/apps.spec.ts

```diff
@@ -52,7 +52,7 @@ describe("Apps class tests", () => {
         const OAUTH2_SCHEME = "OAUTH2";
         expect(inputFields.availableAuthSchemes).toContain(OAUTH2_SCHEME);
         expect(inputFields.authSchemes[OAUTH2_SCHEME].expected_from_user).toEqual(["client_id", "client_secret"]);
-        expect(inputFields.authSchemes[OAUTH2_SCHEME].optional_fields).toEqual(["scopes"]);
+        expect(inputFields.authSchemes[OAUTH2_SCHEME].optional_fields).toEqual(["oauth_redirect_uri", "scopes"]);
```

Comment:
Consider adding a test case that verifies the actual usage of `oauth_redirect_uri` to ensure it's properly handled by the SDK. This would help catch any potential issues with the redirect URI handling.